src/registrar/templates/application_done.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block title %}Thank you for your domain request{% endblock %}
+
+{% block content %}
+<h1>Thank you!</h1>
+
+<p>
+Thank you for your domain request. We'll email a copy of your request to you,
+your authorizing official, and any contacts you added.</p>
+
+<h2>Next steps in this process</h2>
+
+<ul>
+
+    <li> We'll review your request. This could take up to two weeks. During
+        this review we'll verify that you're eligible for a .gov domain, that
+        your authorizing official approves your request, and that your domain
+        meets our naming requirements.
+    </li>
+
+    <li> You can <a href="/application/{{ application_id }}">check the
```

Comment:
Oh, I see what you mean with `/application/id` now.

**Suggestion:** Wrap the `<a>` tags in `<del>` so we have a visual indicator that these are broken/WIP links.